js/plugins/iq.js

```diff
@@ -147,12 +155,31 @@ var IqPlugin= function (framework)
           this.setMetadata(modules.iq);
         }
       }
-      OO.log( "Analytics Template: PluginID \'" + id + "\' received this event \'" + eventName + "\' with these params:", params);
+      return;
+    }
+    //OO.EVENTS.EMBED_CODE_CHANGED -> OO.Analytics.EVENTS.VIDEO_SOURCE_CHANGED.
+    if(eventName === OO.Analytics.EVENTS.VIDEO_SOURCE_CHANGED){ 
+      if (params && params[0] && params[0].metadata)
+      {
+        //autoPlay = params[0].metadata.autoPlay;
```

Comment:
Do we still need this commented line?

Reply:
@aeng7 I uncommented it, as it does seem to be used by INITIAL_PLAYBACK_REQUESTED